spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -87,7 +92,11 @@
 	public FirestoreTemplate(FirestoreStub firestore, String parent) {
 		this.firestore = firestore;
 		this.parent = parent;
-		this.databasePath = parent.substring(0, StringUtils.ordinalIndexOf(parent, "/", 4));
+		this.databasePath = extractDatabasePath(parent);
+	}
+
+	public static String extractDatabasePath(String parent) {
```

Comment:
This should probably be extracted to a utility class just because this method is not meant to be offered to users.

Reply:
done